src/Presentation/Nop.Web/Themes/DefaultClean/Content/css/styles.css

```diff
@@ -1,40 +1,40 @@
-
-
 /*********** CSS RESET **********/
 
-
 * {
-	margin: 0;
-	outline: none;
-	padding: 0;
-	text-decoration: none;
-}
-*, *:before, *:after {
-	-webkit-box-sizing: border-box;
-	-moz-box-sizing: border-box;
-	box-sizing: border-box;
+  margin: 0;
```

Comment:
Ken what are all these changes mean? All the lines in css file were touched. It should be reverted back.

Reply:
That is very strange. I only removed the style that I had added by deleting those lines.  I will just revert the file to the contents in master rather than doing a manual removal. I will do the same on the rtl file.